iohk/NixOps.hs

```diff
@@ -73,6 +75,9 @@ projectURL     IOHK            = "https://github.com/input-output-hk/iohk-nixops
 projectURL     Nixpkgs         = "https://github.com/nixos/nixpkgs.git"
 projectURL     Stack2nix       = "https://github.com/input-output-hk/stack2nix.git"
 
+nixpkgsRevTarball :: Commit -> URL
+nixpkgsRevTarball (Commit rev) = URL $ "https://github.com/NixOS/nixpkgs/archive/"<>rev<>".tar.gz"
```

Comment:
I'm feeling wary of doing anything more involved at this current moment, because then we'd have resolve merge conflicts with `staging`.  The proper way would be to extend the URL data type to allow more structured options, and tie that into defaults and optional args.

But that's going to get problematic during merge conflict resolution.